apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -123,10 +116,10 @@ export default class TeacherHomepage extends Component {
   }
 
   render() {
-    const { courses, topCourse, announcements, queryStringOpen, joinedSections } = this.props;
+    const { courses, topCourse, joinedSections } = this.props;
     const { ncesSchoolId, censusQuestion, schoolYear } = this.props;
     const { teacherId, teacherName, teacherEmail } = this.props;
-    const { canViewAdvancedTools } = this.props;
+    const { canViewAdvancedTools, isEnglish, queryStringOpen } = this.props;
```

Comment:
Out of scope for this change, but just noticed `queryStringOpen` and can't tell what it does from its name :)

Reply:
It's related to managing sections: https://github.com/code-dot-org/code-dot-org/commit/b1111ed88e286c5cfe0b6000548ed686c19fa796